app/controllers/api/v1/todos_controller.rb

```diff
@@ -6,12 +6,12 @@ def index
   end
   def show
     @todo = Todo.find(params[:id])
-    render json: @todos
+    render json: @todo
   end
   def create
     @todo = Todo.new(valid_params)
     if @todo.save
-      render json: @todo, status: 200
+      render json: @todo, status: 201
```

Comment:
I would continue being consistent and use the status symbols.

201 => :created
200 => :ok